addons/website/static/tests/builder/website_builder/image_gallery.test.js

```diff
@@ -44,19 +44,16 @@ test("Add image in gallery", async () => {
         </section>
         `
     );
-    onRpc("/html_editor/get_image_info", () => {
-        expect.step("get_image_info");
-        return {
-            attachment: {
-                id: 1,
-            },
-            original: {
-                id: 1,
-                image_src: "/web/image/hoot.png",
-                mimetype: "image/png",
-            },
-        };
-    });
+    onRpc("/html_editor/get_image_info", () => ({
+        attachment: {
+            id: 1,
+        },
+        original: {
+            id: 1,
+            image_src: "/web/image/hoot.png",
+            mimetype: "image/png",
+        },
+    }));
```

Comment:
why did you remove "expect.step("get_image_info");" ?
Now it s not possible to check if get_image_info is call and how many time ?

Reply:
Maybe rebase your branch, this fix could have an impact https://github.com/odoo/odoo/pull/213185/files